src/controllers/users/getMembersController.ts

```diff
@@ -0,0 +1,30 @@
+import { Response, NextFunction } from 'express';
+import { RequestWithDecoded } from '../../utils';
+import { getMembersQuery } from '../../database';
+
+const getMembersController = async (req: RequestWithDecoded, res: Response, next: NextFunction) => {
+  try {
+    const { page } = req.query;
+
+    const offset = (Number(page || 1) - 1) * 10;
+    const { rows: members } = await getMembersQuery({ offset });
+    const countOfMembers = members.length;
```

Comment:
Try to add 20 users ang get all of them by this controller and let me know about the countOfMembers will return in response